monai/fl/client/monai_algo.py

```diff
@@ -38,32 +39,61 @@ def convert_global_weights(global_weights, local_var_dict):
     model_keys = global_weights.keys()
     for var_name in local_var_dict:
         if var_name in model_keys:
+            weights = global_weights[var_name]
             try:
+                # reshape global weights to compute difference later on
+                weights = torch.reshape(torch.as_tensor(weights), local_var_dict[var_name].shape)
                 # update the local dict
-                local_var_dict[var_name] = torch.as_tensor(global_weights[var_name])
+                local_var_dict[var_name] = weights
             except Exception as e:
                 raise ValueError(f"Convert weight from {var_name} failed.") from e
     return local_var_dict
 
 
+def compute_weight_diff(global_weights, local_var_dict):
+    if global_weights is None:
+        raise ValueError("Cannot compute weight differences if `global_weights` is None!")
+    if local_var_dict is None:
+        raise ValueError("Cannot compute weight differences if `local_var_dict` is None!")
+    # compute delta model, global model has the primary key set
+    weight_diff = {}
+    for name in global_weights:
+        if name not in local_var_dict:
+            continue
+        weight_diff[name] = local_var_dict[name].cpu() - global_weights[name]
+        if torch.any(torch.isnan(weight_diff[name])):
+            raise ValueError(f"Weights for {name} became NaN...")
+    return weight_diff
+
+
 class MonaiAlgo(ClientAlgo):
```

Comment:
Could you please help add the doc-string here update the module level REAME to address ticket: https://github.com/Project-MONAI/MONAI/issues/4845?

Thanks.

Reply:
docstrings are added.